test/opf/sdpwrm.jl

```diff
@@ -67,11 +64,7 @@ This test is executed on the 5 bus system.
 """
 function _test_sdpwrm_DualFeasibility(OPF::Union{Type{PGLearn.SDPOPF}, Type{PGLearn.SparseSDPOPF}})
     T = Float128
-    if OPF == PGLearn.SDPOPF
-        data = PGLearn.OPFData(make_basic_network(pglib("5_pjm")))
-    else
-        data = PGLearn.OPFData(make_basic_network(pglib("14_ieee")))
-    end
+    data = PGLearn.OPFData(make_basic_network(pglib("5_pjm")))
     solver = JuMP.optimizer_with_attributes(Clarabel.Optimizer{T},
```

Comment:
sorry I missed this. Any reason not to use `OPT_SOLVERS[OPF]` here?

Reply:
Here we are using different solver configurations, and it uses `Float128`. It's similar in the SOC test
https://github.com/AI4OPT/PGLearn.jl/blob/6fd494dda2b4ba16a809e7141103b71b238aef5d/test/opf/socwr.jl#L75